common/v2/components/AccountList.tsx

```diff
@@ -30,9 +30,34 @@ import { TUuid } from 'v2/types/uuid';
 const Label = styled.span`
   display: flex;
   align-items: center;
-  @media (min-width: ${BREAK_POINTS.SCREEN_SM} {
+  @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
+    font-weight: bold;
+  }
+`;
+
+const LabelWithWallet = styled.span`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
     font-weight: bold;
-  })
+  }
+`;
+
+const WalletTypeLabelContainer = styled.div`
+  display: block;
+`;
+
+const WalletTypeLabel = styled.p`
```

Comment:
Identicon and content are misaligned.
This is solved if you remove margin-bottom of the paragraph tag.